pkg/gui/pull_request_menu_panel.go

```diff
@@ -0,0 +1,52 @@
+package gui
+
+import (
+	"fmt"
+	"github.com/jesseduffield/lazygit/pkg/commands"
+	"github.com/jesseduffield/lazygit/pkg/commands/models"
+	"github.com/jesseduffield/lazygit/pkg/commands/oscommands"
+)
+
+func (gui *Gui) createPullRequestMenu(selectedBranch *models.Branch, checkedOutBranch *models.Branch) error {
+	menuItems := make([]*menuItem, 0, 2)
+
+	if selectedBranch != checkedOutBranch {
+		menuItems = append(menuItems, &menuItem{
+			displayStrings: []string{
+				fmt.Sprintf("%s -> default branch", selectedBranch.Name),
```

Comment:
we can use the unicode `→` here to make this a little easier on the eyes

Reply:
Fixed